### DISCUSSION TITLE
Harris Patches patched_talkgroups

### DISCUSSION THREAD

**Discussion by kb2ear** [chuot/rdio-scanner#134] [Q&A] [answered]
Do I need to do anything to make harris patches show up in rdio-scanner? They are in the metadata json but I see nothing different on the display nor do I hear anything if the primary talk group is not selected. running version 6.1.14
```
{
"freq": 770593750,
"start_time": 1645914303,
"stop_time": 1645914306,
"emergency": 0,
"encrypted": 0,
"call_length": 2,
"talkgroup": 65197,
"talkgroup_tag": "Patch 65197",
"audio_type": "digital tdma",
"short_name": "mcnj",
"patched_talkgroups": [21301,21410,65197],
"freqList": [{ "freq": 770593750, "time": 1645914303, "pos": 0.0, "len": 2, "error_count": 0.0, "spike_count": 0.0}],
"srcList": [ {"src": 1221002, "time": 1645914303, "pos": 0, "emergency": 0, "signal_system": "", "tag": ""}, {"src": 1221002, "time": 1645914305, "pos": 2, "emergency": 0, "signal_system": "", "tag": ""} ]
}
```

**Comment by chuot** [comment, marked as answer]
Nothing special to do. If `65197` is avoided but one of the `[21301, 21410,65197]` is not avoided, you should see patch instead of avoid on the display. Note that the native version 1.4.6 has a bug in regards to that and which will be fixed in the next native release.
